<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblOrderStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>701</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Order Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddOrder">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompleteOrder">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowOrders">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Pending Orders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowCompletedOrders">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show completed orders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveOrdersToFile">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Completed orders to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextOrder">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display next order </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Restaurant Order Management</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOrderID">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter Order ID to check Status</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCheckOrderStatus">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
